2-0-data-structures-and-algorithms/2-2-7-hash-table/src/test/java/com/bobocode/cs/HashTableTest.java

```diff
@@ -12,22 +15,24 @@
 import java.util.concurrent.ThreadLocalRandom;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-
-import static java.lang.reflect.Modifier.isStatic;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.*;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.ClassOrderer.OrderAnnotation;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestClassOrder;
+import org.junit.jupiter.api.TestMethodOrder;
 
 /**
- * A Reflection-based step by step test for a {@link HashTable} class. PLEASE NOTE that Reflection API should not be used
- * for testing a production code. We use it for learning purposes only!
- *
- * @author Taras Boychuk
- */
+ A Reflection-based step by step test for a {@link HashTable} class. PLEASE NOTE that Reflection API should not be used for testing a
+ production code. We use it for learning purposes only!
+ @author Taras Boychuk */
```

Comment:
@Stanislav-Zabramnyi it seems like you broke the formatting of the javadoc

Reply:
I will revert it